src/mon/NVMeofGwMap.h

```diff
@@ -106,19 +106,19 @@ class NVMeofGwMap
     const NvmeGwId &gw_id, const NvmeGroupKey& group_key,
     NvmeAnaGrpId ANA_groupid, epoch_t &epoch, bool failover);
 
-  void encode(ceph::buffer::list &bl) const {
+  void encode(ceph::buffer::list &bl, uint64_t features) const {
     using ceph::encode;
     ENCODE_START(1, 1, bl);
     encode(epoch, bl);// global map epoch
 
-    encode(created_gws, bl); //Encode created GWs
-    encode(fsm_timers, bl);
+    encode(created_gws, bl, features); //Encode created GWs
+    encode(fsm_timers, bl, features);
     ENCODE_FINISH(bl);
   }
 
   void decode(ceph::buffer::list::const_iterator &bl) {
     using ceph::decode;
-    DECODE_START(1, bl);
+    DECODE_START(2, bl);
```

Comment:
This should be 1, bl -- hold over from the prior set of changes I think.

Reply:
fixed in the last push